src/System.Globalization.Extensions/tests/System.Globalization.Extensions.Tests.builds

```diff
@@ -4,19 +4,17 @@
   <ItemGroup>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
       <OSGroup>Unix</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
-      <TestTFMs>net463</TestTFMs>
```

Comment:
the .csproj inside is setting Windows_NT by default as the OSGroup, which is why it is not needed here. If I keep it as it was it would cause a binclash betweein this build and the default one for netcoreapp1.1, where one has OSGroup set as a global property and the other one is set in the .csproj. That is why I merged them into only one build, that has two testTFMs